src/translations/qterminal_gl.ts

```diff
@@ -555,7 +555,7 @@
     <message>
         <location filename="../forms/propertiesdialog.ui" line="577"/>
         <source>xterm-256color</source>
-        <translation></translation>
+        <translation>xterm en 256 cores</translation>
```

Comment:
@mbouzada This is a special term and should not be translated.

Notes to myself: we should prevent such fields from appearing in `*.ts` files.